pkg/apis/deployment/v1alpha/member_phase.go

```diff
@@ -34,6 +34,8 @@ const (
 	MemberPhaseFailed MemberPhase = "Failed"
 	// MemberPhaseCleanOut indicates that a dbserver is in the process of being cleaned out
 	MemberPhaseCleanOut MemberPhase = "CleanOut"
+	// MemberPhaseDrain indicates that a dbserver is n the process of being cleaned out as result of draining a node
```

Comment:
Suggest to add a helper `func (p MemberPhase) IsCleanOutOrDrain() bool`
will be cleaner in the rest of the code.